src/Microsoft.ComponentDetection.Orchestrator/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -66,7 +66,6 @@ public static IServiceCollection AddComponentDetection(this IServiceCollection s
         services.AddSingleton<IExperimentConfiguration, SimplePipExperiment>();
         services.AddSingleton<IExperimentConfiguration, RustCliDetectorExperiment>();
         services.AddSingleton<IExperimentConfiguration, Go117DetectorExperiment>();
-        services.AddSingleton<IExperimentConfiguration, DotNetDetectorExperiment>();
```

Comment:
In order to promote out of experimental in the detector itself we need to replace `IExperimentalDetector` with `DefaultComponentDetector`

Reply:
I knew I was missing something thanks